python-modules/cis_postgresql/cis_postgresql/execute.py

```diff
@@ -0,0 +1,158 @@
+"""Allows the execution of complex queries against postgres."""
+import operator
+import sqlalchemy
+import unicodedata
+from cis_identity_vault.models import rds
+from sqlalchemy.dialects.postgresql import JSON, TEXT
+
+def attr_single_or_multi(attr):
+    """[return value or values for an attribute for use in queries]
+    
+    Arguments:
+        attr {string} -- [name of the attribute]
+    """
+
+    values = [
```

Comment:
other pieces of code just try if its values vs value. either way it works, just wanted to mention it